README.md

```diff
@@ -654,15 +614,15 @@ Enabled by `ctrlKeysForInsertMode`
 
 Default: `["a", "d", "h", "j", "o", "r", "t", "u", "w"]`
 
-Refer to VIM's manual for their use.
+> 💡 Run the `Preferences: Open Keyboard Shortcuts` vscode command and search for "neovim" to see all keybindings.
```

Comment:
That will only tell you that it's passed down to neovim, the meaning of the actual binding lives in the vim manual.

Reply:
can add another note to mention that.

but in general: why is someone using a vim extension if they don't know how to use vim?

Reply:
I think I am an example of that - I was using VS code, and then wanted to learn how to use vim. However, I missed a lot of VScode's features, like out of the box LSP and the UI. So actually, I learned how to use vscode-neovim before I learned how to use terminal vim, so the reference actually acted as a convenient cheat sheet for me.

Reply:
will add a note that mentions `:help` and also links to relevant parts of https://neovim.io/doc/user/index.html